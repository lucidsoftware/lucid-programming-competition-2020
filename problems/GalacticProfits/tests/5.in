64
10
Random0
68 23 9 80 96 66 80 39 87 36 12 69 25 46 16 39 24 67 7 53 35 8 57 60 84 56 54 34 55 94 37 39 31 59 76 83 64 25 27 8 50 17 43 70 70 49 60 79 19 44 38 36 84 98 92 25 12 9 96 68 15 70 73 27
Random1
89 61 66 1 39 46 77 43 32 43 83 92 10 17 39 3 57 18 61 48 74 10 51 19 74 41 28 67 3 79 9 34 37 53 91 70 70 48 25 24 55 65 21 6 54 19 40 58 66 15 18 2 76 78 8 34 29 65 82 98 66 46 59 39
Random2
63 55 35 86 31 3 67 77 18 3 31 8 43 33 12 2 79 38 5 99 83 60 96 13 1 63 89 60 77 1 26 86 83 46 36 41 48 87 4 17 24 67 45 56 39 56 76 49 11 60 39 84 79 94 50 83 74 78 69 96 28 21 14 49
Random3
73 69 38 52 41 80 32 2 26 73 77 45 25 67 40 18 78 22 44 85 51 4 18 88 61 7 37 13 18 73 23 60 74 4 2 11 3 75 14 80 48 1 23 88 46 49 90 76 35 1 55 63 66 33 37 88 22 36 72 11 30 73 93 30
Random4
57 90 89 61 54 84 28 12 47 58 10 87 14 56 55 41 82 84 12 77 47 39 36 77 85 78 57 62 49 68 38 45 55 3 27 16 2 32 61 18 90 39 13 43 12 48 60 14 29 58 57 12 16 2 45 27 44 1 79 7 73 78 54 38
Random5
54 33 17 79 3 53 21 83 40 97 32 62 6 97 34 9 37 44 68 45 9 47 5 72 15 11 77 63 90 84 65 61 6 77 34 75 26 12 23 75 49 29 9 66 83 33 86 58 58 43 94 69 84 36 21 44 88 52 68 32 16 2 24 9
Random6
89 43 50 3 59 68 68 25 25 16 56 84 74 54 58 48 10 17 81 72 63 66 44 29 23 82 62 71 26 83 46 70 44 40 44 29 96 4 82 60 17 98 81 67 19 72 58 37 62 93 53 32 74 65 95 74 38 70 60 69 31 54 2 83
Random7
66 15 88 3 93 25 21 92 5 10 57 81 93 48 55 91 2 15 42 55 95 3 88 93 61 8 52 35 85 73 82 38 19 94 68 87 66 20 18 22 79 75 39 64 33 46 75 99 48 60 16 51 80 3 81 90 95 33 32 33 75 66 37 58
Random8
7 49 73 25 22 9 3 66 47 9 96 29 29 98 35 8 15 54 56 28 92 57 2 57 45 43 28 84 97 47 37 81 92 5 43 61 6 68 45 38 64 1 56 58 55 95 22 65 7 91 45 20 29 18 37 91 9 99 55 54 28 35 63 7
Random9
80 47 9 18 63 13 29 78 43 67 90 96 57 1 39 47 57 55 97 64 63 26 83 4 78 60 84 31 30 53 89 75 80 70 82 89 92 50 18 17 62 77 56 71 52 53 81 15 29 69 13 23 23 82 26 7 36 14 25 13 76 15 4 35
