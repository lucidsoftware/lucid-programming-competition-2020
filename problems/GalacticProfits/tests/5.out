Max: 3309052
Min: -265222